### PR TITLE
fix(themes): THEME-989 cart options fixes & enhancements

### PR DESCRIPTION
**Pull request by Ahmed-Basalib10**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* Fix

What is the current behaviour? (You can also link to an open issue here)

* Option price for numbers and text fields not showing in order page & options cost not showing in cart page

What is the new behaviour? (You can also link to the ticket here)

* Show Option price for numbers and text fields in order page & show options cost in cart page

Does this PR introduce a breaking change?

* No

Screenshots (If appropriate)

*

### CODE REVIEW

File: src/locales/ar.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
   },
   "pages": {
     "cart": {
-      "cart_options": "خيارات السلة"
+      "cart_options": "خيارات السلة",
+      "cart_options_cost": "تكلفة خيارات الطلب"
```
Comment (jalmatari) [inline src/locales/ar.json:13]: use `pages.cart.order_options_total`

**Review by jalmatari — CHANGES_REQUESTED** [review]

**Comment by SallaDev** [comment]
**Approve:**
- can not merge pr (https://github.com/SallaApp/theme-raed/pull/377) needs at least `one approve ❗️`



Merge by: @jalmatari

**Review by jalmatari — APPROVED** [review]

**Comment by jalmatari** [comment]
merge

**Comment by SallaDev** [comment]
https://github.com/SallaApp/theme-raed/pull/377 is been merged ✅

-------------------

Merge by: @jalmatari

**Pull request merged**